### DISCUSSION TITLE
TypeError: 'type' object is not subscriptable

### DISCUSSION THREAD

**Discussion by good-demo** [pyg-team/pytorch_geometric#5284] [Q&A]
Hello,everyone, have you encountered this problem?

class DynamicBatchSampler(torch.utils.data.sampler.Sampler[List[int]]):
TypeError: 'type' object is not subscriptable

Evironment:
torch                 1.4.0+cu100
torch-cluster         1.5.2
torch-geometric       2.1.0
torch-scatter         2.0.4
torch-sparse          0.5.1
torch-spline-conv     1.2.0

**Comment by rusty1s** [comment]
See https://github.com/pyg-team/pytorch_geometric/issues/5285
